docs/references/kontena-yml.md

```diff
@@ -467,21 +482,23 @@ web:
     - "${EXTERNAL_PORT}:5000"
 ```
 
-Referencing other services within the same application requires the project prefix in the service name. You can use `${project}` variable for this:
+Kontena will define automatically `GRID` and `STACK` environment variables for you and those are available to use for variable substitution.
 
-```
-web:  
-  image: wordpress:latest
-  environment:
-    - WORDPRESS_DB_HOST=${project}-db
-db:
-  image: mariadb:latest
-```
+You can also define variables and use values from those for variable substitution. See the complete [variables reference](kontena-yml-variables.md)
 
 ## Example kontena.yml
 
 ```
-version: "2"
+stack: kontena/example-app
+version: 0.1.0
+variables:
+  mysql_root_pw:
+    type: string    
+    from:      
+      prompt: Enter a root password for MySQL or leave empty to auto generate
+      random_string: 16
+    to:
```

Comment:
More succient example would be to just omit this `to: ...` and use `${mysql_root_pw}` instead.